app/controllers/application_controller.rb

```diff
@@ -144,6 +144,21 @@ def api_doc_definition
     self.class.send(:api_doc_definition)
   end
 
+  def messaging_client
+    require "manageiq-messaging"
+
+    @messaging_client ||= ManageIQ::Messaging::Client.open(messaging_opts)
+  end
+
+  def messaging_opts
```

Comment:
I know this was just a copy & paste but since these are constant and only ever used by https://github.com/ManageIQ/topological_inventory-api/pull/150/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR150 can you just put this in the `messaging_client` method like
```
@messaging_client ||= ManageIQ::Messaging::Client.open({
  :protocol => :kafka,...
})
```

Reply:
👍